.github/workflows/CI-CD.yml

```
@@ -0,0 +1,44 @@
+name: Build, publish and deploy to MonsterASP.NET
+on: [push]
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)